app/Http/Controllers/Admin/StatisticsController.php

```diff
@@ -29,7 +29,7 @@ public function index(Request $request)
         $servers = Server::all();
         $nodes = Node::all();
         $serversCount = count($servers);
-        $nodesCount = Node::count();
+        $nodesCount = count($nodes);
```

Comment:
I'm not sure these counts are necessary, you should be able to do `{{ count($servers) }}` in the blade templates, or `Pterodactyl.servers.length` in JS and keep things a little cleaner. :)